src/view/components/decisionScopesComponent.jsx

```diff
@@ -0,0 +1,115 @@
+import "regenerator-runtime"; // needed for some of react-spectrum
+import React from "react";
+import { FieldArray } from "formik";
+import Button from "@react/react-spectrum/Button";
+import "@react/react-spectrum/Form"; // needed for spectrum form styles
+import PropTypes from "prop-types";
+import Textfield from "@react/react-spectrum/Textfield";
+import FieldLabel from "@react/react-spectrum/FieldLabel";
+import Delete from "@react/react-spectrum/Icon/Delete";
+import RadioGroup from "@react/react-spectrum/RadioGroup";
+import Radio from "@react/react-spectrum/Radio";
+import InfoTipLayout from "./infoTipLayout";
+import WrappedField from "./wrappedField";
+
+function DecisionScopesComponent({ values, formikProps, options }) {
+  return (
+    <div>
+      <div className="u-gapTop">
+        <InfoTipLayout
+          tip="If you set decision scopes as a Data Element. That Data Element should return an array.
+        Otherwise you could add decision scopes as constants."
+        >
+          <FieldLabel
+            labelFor="decisionScopesType"
+            label="Decision Scopes (optional)"
+          />
+        </InfoTipLayout>
+        <WrappedField
+          id="decisionScopesTypeField"
+          name="option"
+          component={RadioGroup}
+          componentClassName="u-flexColumn"
+        >
+          <Radio
+            data-test-id="constantOptionField"
+            value={options.CONSTANT}
+            label="Constant"
```

Comment:
Should we rename this to `Enter a value`? Or something similar? `Constant` sounds too technical. @Aaronius @jonsnyder ?

Reply:
`enter a value` looks better